src/deepsparse/server/config.py

```diff
@@ -144,16 +149,36 @@ class ServerConfig(BaseModel):
 
     endpoints: List[EndpointConfig] = Field(description="The models to serve.")
 
-    loggers: Union[Dict[str, Dict[str, Any]], str, None] = Field(
+    loggers: Optional[List[Union[str, Dict[str, Dict[str, Any]]]]] = Field(
         default=None,
         description=(
             "Optional dictionary of logger integration names to initialization kwargs."
-            " Set to 'default' for default logger based on deployment. Set to None for"
-            " no loggers. Default is `None`. Example: "
-            "{'prometheus': {'port': 8001}}."
+            "Set to None for no loggers. Default is `None`."
         ),
     )
 
+    @validator("endpoints")
+    def set_unique_endpoint_names(
```

Comment:
why are we adding this here - seems orthogonal to the purpose of this PR, additionally don't we already catch this as a user error?

Reply:
I agree and disagree:
1. Yes, it is orthogonal to this PR. It makes more sense to add it to this PR's successor.
2. Currently are catching the error yes, but only after the loggers have been created. This check should happen before the loggers are created, and IMO the best place for it is on the instantiation of the BaseModel.

I will remove this feature for now, but will revisit in the next PR.